lib/internal/Magento/Framework/Data/Collection.php

```diff
@@ -492,7 +492,7 @@ public function clear()
      *
      * Returns array with results of callback for each item
      *
-     * @param string $callback
+     * @param string|array $callback
```

Comment:
Please replace to `callable`, I believe it is a better alternative.

As always, amend commit and do force push, please.